docker-app/qfieldcloud/core/admin.py

```diff
@@ -568,6 +568,27 @@ class ProjectFilesWidget(widgets.Input):
     template_name = "admin/project_files_widget.html"
 
 
+class OwnerTypeFilter(admin.SimpleListFilter):
+    # "owner__type" for the filter would suffice,
+    # But the filter title would then be "By type" in the UI,
+    # which is semantically unclear, hence the SimpleListFilter subclass
```

Comment:
I think comment is not required, as we already have done this in other `SimpleListFilter` instances. It's expected behaviour from Django.